tezos/python-tests/daemons/baker.py

```diff
@@ -46,18 +46,14 @@ def __init__(
         if run_params is None:
             run_params = []
         endpoint = f'http://127.0.0.1:{rpc_port}'
-        cmd = [baker, '-m', 'json', '-base-dir', base_dir, '-endpoint', endpoint]
+        cmd = [baker, '-m', 'json']
         cmd.extend(params)
-        cmd.extend(
-            [
-                'run',
-                'with',
-                'local',
-                'node',
-                node_dir
-            ]
-            + accounts
-        )
+        if os.environ.get("RUN_TEZEDGE_BAKER") == "external":
+            cmd.extend([baker, '--base-dir', base_dir, '--endpoint', endpoint, '--archive'])
```

Comment:
@vlad9486 the initial `baker` here is not needed right? it is already part of the original `cmd` list

Reply:
@tizoc  you are right